core/lib/protobuf_config/src/proto/config/da_client.proto

```diff
@@ -37,9 +37,24 @@ message CelestiaConfig {
   optional uint64 timeout_ms = 4;
 }
 
+message Url {
+  optional string g1_url = 1;
+  optional string g2_url = 2;
+}
+
 message EigenConfig {
-  optional string rpc_node_url = 1;
-  optional uint64 inclusion_polling_interval_ms = 2;
+  optional string disperser_rpc = 3;
+  optional uint32 settlement_layer_confirmation_depth = 4;
+  optional string eigenda_eth_rpc = 5;
+  optional string eigenda_svc_manager_address = 6;
+  optional bool wait_for_finalization = 7;
+  optional bool authenticated = 8;
+  oneof points_source {
+    string path = 9;
+    Url url = 10;
+  }
```

Comment:
How will this be represented in YAML? AFAIU, the `path` or `url` would be directly included into the config object w/o `points_source` being present anywhere, which is somewhat confusing (which `path` / `url` are we talking about?). I'd suggest to add a prefix for these 2 fields (e.g., `points_source_path` / `points_source_url`).